inlong-common/src/main/java/org/apache/inlong/common/pojo/agent/DataConfig.java

```diff
@@ -74,6 +75,13 @@ public class DataConfig {
     private DataProxyTopicInfo topicInfo;
 
     public boolean isValid() {
-        return true;
+        if (dataReportType == 0 || dataReportType == 1) {
```

Comment:
Suggest extracting a constant class or enum to reuse the data report type.

Please search the following constant in the manager module:

```
public static final Integer REPORT_TO_DP_RECEIVED = 0;
```